practitioner/src/main/java/org/smartregister/extension/rest/PractitionerDetailsResourceProvider.java

```diff
@@ -346,6 +356,26 @@ private List<Organization> mapToTeams(List<IBaseResource> teams) {
         return organizations;
     }
 
+    private List<PractitionerRole> mapToPractitionerRoles(List<IBaseResource> practitionerRoles) {
+        List<PractitionerRole> practitionerRoleList = new ArrayList<>();
+        PractitionerRole practitionerRoleObj;
+        for (IBaseResource practitionerRole : practitionerRoleList) {
+            practitionerRoleObj = (PractitionerRole) practitionerRole;
+            practitionerRoleList.add(practitionerRoleObj);
+        }
+        return practitionerRoleList;
+    }
```

Comment:
@rehammuzzamil the `practitionerRoles` param doesn't seem to be used. This means the `practitionerRoleList` will always a size of 0.